gems/pending/util/vcenter_thumb_print.rb

```diff
@@ -0,0 +1,25 @@
+require 'util/thumb_print'
+
+class VcenterThumbPrint < ThumbPrint
```

Comment:
esx_thumb_print & vcenter_thumb_print moved to new gems/pending/vmware directory.